server/config/toml.go

```diff
@@ -165,6 +165,12 @@ retries = {{ .Rosetta.Retries }}
 # Offline defines if Rosetta server should run in offline mode.
 offline = {{ .Rosetta.Offline }}
 
+# SuggestGas defines gas limit when calculate fee
+suggest-gas = {{ .Rosetta.SuggestGas }}
+
+# DefaultSuggestDenom defines the defult denom for fee suggestion
+default-suggest-denom = "{{ .Rosetta.DefaultSuggestDenom }}"
+
```

Comment:
is there a way we can use the auto gas functionality to populate the suggest field. This could change as the chain grows, as we've seen with 0.42 of the sdk. 

The default could also come from the app instead of the config, this way if the chain changes some parameters it could suggest a new default

Reply:
suggested_fee is in result of [construction/metadata](https://www.rosetta-api.org/docs/ConstructionApi.html#constructionmetadata). i don't think we can calculate gas automatically because there's no information about transaction in request..

if options in request contains gas_limit, it will be used to suggest fee instead of the config.

Reply:
is it an issue if this can be set to anything, but on chain it could be 2x more.

Reply:
@marbar3778 

Can you elaborate your question? 

You mean setting the suggest-gas to anything?? 
I can be problem, the rosetta client will use this default gas as its gas limit. 

hmm maybe we can use fixed gas limit like 200,000, which is used in default gas in cmd client. 

+ just to clarify the problem,
When we return this `suggest-gas` there is no information like which tx the client want to use or how long the tx size will be. It's just return some fixed gas limit.

Reply:
Or we can use client default gas 200,000 which is used in tx command.

Reply:
@marbar3778 any suggestion for this?

Reply:
lets use the 200_000 amount for now. for the time being this is the best we can provide until we have a cleaner solution.

Reply:
option to disable default fee suggestion added